sdk/src/client/node_api/core/routes.rs

```diff
@@ -103,6 +103,18 @@ impl ClientInner {
             .await
     }
 
+    /// Returns the totally available Mana rewards of an account or delegation output decayed up to epochEnd index
+    /// provided in the response. GET /api/core/v3/rewards/{outputId}
+    pub async fn get_rewards(&self, output_id: &OutputId) -> Result<ManaRewardsResponse> {
```

Comment:
Just a suggestion, for the congestion route I've slightly edited the name from the TIP to make it a bit less generic, maybe do the same? `get_mana_rewards`? `get_output_rewards`?